README.md

```diff
@@ -213,6 +213,9 @@ for (sObjectField field : MyTriggers.getChangedFields(fieldsToCheck,record,recor
 // add all records in the current update context
 MyTriggers.addUpdatedIds(triggerOldMap.keySet());
 
+// if the recursion control should be limited to a specific trigger handler class, use:
+this.addUpdatedIdsForType(triggerOldMap.keySet());
```

Comment:
`this` => `MyTriggers`

Reply:
My method is a class method, not a static method. Thus, it needs to be called with the current instance as receiver (using this.).
The reason that it is a class method is that it must hold IDs tied to a specific instance.

Reply:
I must have missed the instance vs static part, I am sorry. Need to meditate on this a little further.